packages/compartment-mapper/package.json

```diff
@@ -38,7 +38,8 @@
     "lint:js": "eslint .",
     "lint:types": "tsc -p jsconfig.json",
     "prettier-fixtures": "prettier --write --with-node-modules './test/fixtures-*/**/*.*js'",
-    "test": "ava"
+    "test": "ava",
+    "dev:livebundle": "mkdir -p dist && nodemon scripts/bundle-live-test.js --ignore dist/bu.js"
```

Comment:
example bundle is no longer being committed, but I kinda regret that. It'd show what changed in the PR really well.

Reply:
snapshot tests!